pkg/reconciler/intevents/reconciler.go

```diff
@@ -175,15 +189,101 @@ func (psb *PubSubBase) ReconcilePullSubscription(ctx context.Context, pubsubable
 		}
 	}
 
+	// TODO Compare the actual PullSubscription we found with the desired one and update as needed.
```

Comment:
is this TODO still relevant?

Reply:
Nope :)

Bad merge with your change that just added this.